packages/experiments/src/components/Chiclet/Chiclet.types.ts

```diff
@@ -37,6 +41,36 @@ export interface IChicletProps extends React.Props<ChicletBase> {
    */
   description?: React.ReactElement<JSX.Element>;
 
+  /**
+   * Image to render for the component.
+   */
+  image?: string;
+
+  /**
+   * Image Width to render for the component.
+   */
+  imageWidth?: string;
+
+  /**
+   * Image Height to render for the component.
+   */
+  imageHeight?: string;
+
+  /**
+   * Alternate image to render for the component.
+   */
+  imageAlt?: string;
+
+  /**
+   * Icon to render for the component.
+   */
+  itemType?: string;
```

Comment:
If this is a known set of file extensions you might consider typing it as a string [union type](https://www.typescriptlang.org/docs/handbook/advanced-types.html#union-types):

```
itemType?: 'docx' | 'ppt' | 'xlsx' ...
```

Reply:
Yes, it is a known set